src/amoc_config_validation.erl

```diff
@@ -8,4 +8,9 @@ positive_integer(I) -> is_integer(I) andalso I > 0.
 
 nonnegative_integer(I) -> is_integer(I) andalso I >= 0.
 
-binary(Bitstring) -> is_binary(Bitstring).
+binary(Binary) -> is_binary(Binary).
```

Comment:
As I see this function is used only inside of `binary_or_undefined/1` function, maybe make sense remove this function and use only `is_binary/1` in `binary_or_undefined/1`.

Reply:
These functions are used to validate the parameters of scenarios and scenario helpers.